content/en/docs/contribute/localization.md

```diff
@@ -119,11 +136,9 @@ mkdir -p content/de/docs/tutorials
 cp content/en/docs/tutorials/kubernetes-basics.md content/de/docs/tutorials/kubernetes-basics.md
 ```
```

Comment:
Maybe add something like: Some editors offers plugins to quickly translate text. For instance VSCode with Google translate plugin. Those tools can really help but shouldn't be used blindly because their translation are not always accurate.

Reply:
now that i didn't know, thanks for the tip @remyleone 👍

Reply:
I'm happy to add a note, but I'd like to stay agnostic to specific products. They sometimes go stale faster than content. Otherwise 👍

Reply:
I added this:
```
Translation tools can speed up the translation process. For example, some editors offers plugins to quickly translate text. 

{{< caution >}}
Use translation tools with care. Their output is not always accurate.
{{< /caution >}}
```